ProffieOS.ino

```diff
@@ -1646,7 +1666,11 @@ SSD1306 display;
 #ifdef ENABLE_MOTION
 
 #include "motion/motion_util.h"
-#include "motion/mpu6050.h"
+#ifdef USE_TEENSY4
+ #include "motion/mpu6050async.h"
+#else
+ #include "motion/mpu6050.h"
```

Comment:
OK,
 I have looked into making the 'normal' and 'async' reading of the mpu6050 both into the mpu6050,h  file, and have an #if defined(__IMXRT1062__) or #ifdef USE_TEENSY4 switch between the two implementations. The differences are quite big between the two, but would allow to use just 1 include for the mpu6050?
Would that be a better idea?